### ISSUE TITLE
Please change document tutorial with javascript code not coffescript.

### ISSUE THREAD

**Issue by lyquocnam** [Lepozepo/cloudinary#96]
Dear author, i new so i hard to read coffee script. i think we should change to javascript or with es6.
it will be easier for new commer.

thanks author for good package !

**Comment by NitroBAY** [comment]
Yes please, it's hard and frustrating to try to read coffee. Just use common or ES6 js. Use this website if you need help http://js2coffee.thomaskalka.de

**Comment by szimmers** [comment]
try this:
http://decaffeinate-project.org/repl/
